### PR TITLE
Add new project icons

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes https://github.com/thenewboston-developers/Website/issues/1729

Added two icons required in the weekly progress design.

### CODE REVIEW

File: src/components/Icon/custom/MilestonesIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+          <path
+            d="M20.6391 22.712L21.8192 23.9093C21.9015 23.9928 22.0094 24.0346 22.1172 24.0346C22.2251 24.0346 22.3329 23.9928 22.4152 23.9093L24.6066 21.6858C24.7712 21.5188 24.7712 21.2481 24.6066 21.081C24.442 20.9141 24.1752 20.9141 24.0106 21.081L22.1172 23.0022L21.2351 22.1071C21.0704 21.9402 20.8037 21.9402 20.639 22.1071C20.4745 22.2742 20.4745 22.545 20.6391 22.712Z"
+            fill="white"
+          />
+        </g>
+        <path
+          d="M31.6694 23.5738H25.1057C24.7536 23.2761 25.3493 22.6006 25.971 22.5552C26.2413 22.535 26.5825 22.5913 26.9729 22.6734C27.0632 21.4769 27.5786 16.0994 29.5232 16.3157C31.7091 16.5579 29.5232 18.6836 28.9051 19.9429C28.4475 20.8757 27.9646 22.2128 27.7442 22.8436C27.9213 22.8818 28.1056 22.9192 28.2928 22.9538C28.5827 22.0665 29.6605 19.083 31.0311 19.2567C32.6409 19.46 31.9939 20.6565 30.8532 21.3666C30.0162 21.8884 28.989 22.6309 28.5003 22.9892C28.9138 23.0562 29.346 23.1016 29.7841 23.0915C30.9082 23.0656 31.4568 23.3964 31.6694 23.5738Z"
+          fill="#82B378"
+        />
+        {state === 'default' && (
```
Comment (jamessspanggg) [inline src/components/Icon/custom/MilestonesIcon.tsx:79]: currently the `state` is used here, where `default` will add the rectangle to create the `not selected` effect while `active` and `hover` will render the original svg

Screen Shot 2021-04-18 at 4 27 22 PM

**Comment by jamessspanggg** [comment]
@angle943 seems like i do not have permissions to seek for reviews, so tagging you here.

**Comment by angle943** [comment]
@jamessspanggg gave you access. i'll also review this after i get through all my notifications

**Comment by angle943** [comment]
Can we move these icons out of the icon file, and make a separate component out of it? Perhaps `ProjectIcons` or something? I feel like these icons are different enough from our other icons (such as colors, etc) that we shouldn't mix the two components together

**Comment by jamessspanggg** [comment]
> Can we move these icons out of the icon file, and make a separate component out of it? Perhaps `ProjectIcons` or something? I feel like these icons are different enough from our other icons (such as colors, etc) that we shouldn't mix the two components together

sure, will do that

**Pull request merged**